data-mongodb/src/main/java/io/micronaut/data/mongodb/conf/MongoDataConfiguration.java

```diff
@@ -18,19 +18,17 @@
 import io.micronaut.context.annotation.ConfigurationProperties;
 import io.micronaut.context.condition.Condition;
 import io.micronaut.context.condition.ConditionContext;
+import io.micronaut.core.annotation.ReflectiveAccess;
 
 /**
  * Micronaut Data MongoDB integration configuration.
  */
 @ConfigurationProperties(MongoDataConfiguration.PREFIX)
```

Comment:
No idea why we need this now, but without mongo native tests are failing because no bean of `MongoDataConfiguration` is found and custom conditions using it are failing.
@dstepanov